modules/cli-administrator-cluster-management.adoc

```diff
@@ -5,10 +5,29 @@
 [id="cli-cluster-management-commands_{context}"]
 = Cluster management CLI commands
 
+== inspect
+
+Gather debugging information for a particular resource.
+
+[NOTE]
+====
+This command is experimental and might change without notice.
+====
+
+.Example: Collect debugging data for the OpenShift API server cluster Operator
+----
+$ oc adm inspect clusteroperator/openshift-apiserver
```

Comment:
Should we highlight or make a note, about the resource you can pass to this? 

@soltysh having a user pass a namespace and a cluster operator will cause issues and possible support cases.